src/network/providers/solana.ts

```diff
@@ -1,11 +1,15 @@
 import * as web3 from '@solana/web3.js';
+import {getHashedName, getNameAccountKey, NameRegistryState} from '@solana/spl-name-service';
 import {mnemonicToSeedSync} from 'bip39';
 import {getSecretPhrase} from '../../wallet/keystore';
 import config from 'config';
 import axios from 'axios';
 const logger = require('../../core/log');
 const log = logger.child({module: 'SolanaProvider'});
 
+// Address of the `.sol` TLD
+const SOL_TLD_AUTHORITY = new web3.PublicKey('58PwtjSDuFHuUkYjH9BYnnQKHfwo9reZhC2zMJv9JPkx');
```

Comment:
Minor: I would move the address to config. Even if the address is not going to change often, I think it would be easier to find it there. BTW is it the same address in both main net and test net?

Reply:
Yes, it's the same address.

Ok, I'll move it to config and add it in the _default.yaml_ file.